mmv1/third_party/terraform/services/secretmanagerregional/resource_secret_manager_regional_secret_version_test.go.erb

```diff
@@ -0,0 +1,165 @@
+<% autogen_exception -%>
+package secretmanagerregional_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccSecretManagerRegionalRegionalSecretVersion_import(t *testing.T) {
```

Comment:
Out of curiosity, why do we need this test specifically, I don't see it's different from the first step of `TestAccSecretManagerRegionalRegionalSecretVersion_update`

Reply:
I included this test case to maintain consistency with the other resources and to test a basic import scenario. However, I understand your point about the redundancy, so I've decided to remove this test case.